packages/suite/src/views/wallet/coinmarket/common/CoinmarketIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+
+import { iconSizes, Image, Row } from '@trezor/components';
+import { borders } from '@trezor/theme';
+
+import { useSelector } from 'src/hooks/suite';
+
+const CoinmarketIconWrapper = styled.div<{ $isDark: boolean }>`
+    ${({ $isDark }) => $isDark && `background-color: #fff;`}
```

Comment:
It's just my opinion, but I would suggest to overlay colors of all these transparent pngs. It would be much more consistent with Suite design.

Something like:

```css
img {
  filter: brightness(0) invert(1);
}
```

or another solution like this one: https://bdwm.be/css-change-a-transparent-png-to-any-color-you-want/

Reply:
We do not want to change the partner's logos.